### PR TITLE
API4 Testings /not finished yet

### PR DESCRIPTION
**Pull request by suerion**
Not finished

### CODE REVIEW

**Comment by suerion** [comment]
Updated from Austins Prebuild without GUI, Updated some Typo Errors, 

GUI an Issues are next

File: src/MihaiChirculete/WorldGuard/WorldGuard.php [hunk 1]
```diff
@@ -223,31 +225,36 @@ public function getRegionNameFromPosition(Position $pos) : string
...
         }
-        if($currentRegion == ""){
-            if ($this->regionExists("global.".$pos->getLevel()->getName())){
-                $currentRegion = "global.".$pos->getLevel()->getName();
+        if ($currentRegion == "") {
+            if ($this->regionExists("global." . $pos->getWorld()
+                ->getDisplayName())) {
+                $currentRegion = "global." . $pos->getWorld()->getDisplayName();
             }
         }
```
Comment (PSYCHO678) [inline src/MihaiChirculete/WorldGuard/WorldGuard.php:245]: you need to swap line 246 with 246

**Review by PSYCHO678 — APPROVED** [review]
this file works

**Comment by suerion** [comment]
Added Last changings from @awgybop1

**Pull request merged**